ts-rs/src/serde_json.rs

```diff
@@ -0,0 +1,21 @@
+use std::collections::HashMap;
+
+use super::{impl_primitives, impl_shadow, TS};
+
+#[derive(TS)]
+#[ts(
+    crate = "crate",
+    rename = "Value",
```

Comment:
Last thing I'm unsure about here: Is `Value` to vague? We currently don't/can't deal with name collisions, so maybe `JsonValue` might be a better name.

Reply:
`JsonValue` would be a better name, but given the use of `#[ts(export_to = "serde_json/")]` I think a name collision is really unlikely

Reply:
Alright, lemme quickly rename it. The name collision i'm more worried about is 
```ts
import type { Value } from "../serde_json/Value";
import type { Value } from "../somewhere/else/Value";
```

Reply:
Oh, I see. I though you were worried about the file getting overwritten